src/app.js

```diff
@@ -93,14 +93,23 @@ function init() {
   };
 
   if (process.env.NODE_ENV !== "development") {
+    const allowedIps = [];
+
+    if (process.env.INDIVIDUAL_ALLOWED_IPS) {
+      process.env.INDIVIDUAL_ALLOWED_IPS.split(" ").forEach((ip) =>
+        allowedIps.push(ip)
+      );
+    }
+
+    if (process.env.ALLOWED_IP_RANGES) {
+      process.env.ALLOWED_IP_RANGES.split(" ").forEach((ipRange) => {
+        allowedIps.push([ipRange.split("-")[0], ipRange.split("-")[1]]);
```

Comment:
You could maybe just do `allowedIps.push(ipRange.split("-"));` (if you're confident you formatted it correctly).

Reply:
Alternately, maybe just have a single environment variable that is the array and call eval on it.
ALLOWED_IP_ADDRESSES=['xxx.xxx.xxx.xxx', ['xxx.xxx.xxx.xxx', 'yyy.yyy.yyy.yyy']]

Reply:
Eh I feel weird using eval when there's an ok alternative solution. Especially since people not familiar with the codebase have permissions to update those variables.